### PR TITLE
[FLINK-13154][docs] Fix broken links of web docs

### PR DESCRIPTION
**Pull request by Myasuka**
## What is the purpose of the change

Fix Flink's broken link before release-1.9

## Brief change log

Fix Flink's broken link before release-1.9

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): **no**
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: **no**
  - The serializers: **no**
  - The runtime per-record code paths (performance sensitive): **no**
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: **no**
  - The S3 file system connector: **no**

## Documentation

  - Does this pull request introduce a new feature? **no**
  - If yes, how is the feature documented? **not applicable**

### CODE REVIEW

**Comment by Myasuka** [comment]
@flinkbot attention @bowenli86 for I have removed one line in `sqlClient.md`.

File: docs/dev/table/sqlClient.md [hunk 1]
```diff
@@ -456,8 +456,6 @@ catalogs:
 
 Currently Flink supports two types of catalog - `FlinkInMemoryCatalog` and `HiveCatalog`.
 
-For more information about catalog, see [Catalogs]({{ site.baseurl }}/dev/table/catalog.html).
```
Comment (bowenli86) [inline docs/dev/table/sqlClient.md]: please revert this line. catalog doc is going to be added in https://github.com/apache/flink/pull/8976

Comment (Myasuka) [reply docs/dev/table/sqlClient.md]: If #8976 would be merged before Flink-1.9 release, I would wait for your PR merged and rebase code then.

Comment (bowenli86) [reply docs/dev/table/sqlClient.md]: yes. it's gonna take a while though

Comment (zentol) [reply docs/dev/table/sqlClient.md]: If it's gonna be a while then we should merge the PR as is and add this line in #8976.

Comment (Myasuka) [reply docs/dev/table/sqlClient.md]: As discussed with @bowenli86 offline, we would first let this PR merged.

Comment (carp84) [reply docs/dev/table/sqlClient.md]: +1 for making this in asap since it's a blocker for 1.9.0 release.

**Pull request merged**